_news/172.md

```diff
@@ -0,0 +1,33 @@
+---
+id: 172
+layout: post
+title: "Best Practice: Reporting a Security Issue in Erlang/OTP"
+lead: "Best Practice: Reporting a Security Issue in Erlang/OTP"
+tags: "Erlang, OTP, security, report, bug"
+date: "2024-10-07"
+author: "Kiko Fernandez-Reyes"
+---
+
+Best Practice: Reporting a Security Issue in Erlang/OTP
+
+## TLDR
+Please **do not create a public issue** for a security issue, use instead [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security).
+
+Please follow this document in order to report security vulnerabilities in Erlang/OTP. 
+
+## When should you report a security issue?
+
+The risk level is often determined by a product of the impact once exploited, and the probability of exploitation occurring. In other words, if a bug can cause great damage, but it takes highest privilege to exploit the bug, then the bug is not a high risk one. Similarly, if the bug is easily exploitable, but its impact is limited, then it is not a high risk issue either.
+
+There is not any hard and fast rule to determine if a bug is worth reporting as a security issue to [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). A general rule is that a bug which allows an unprivileged user to successfully attack the Erlang application, the Erlang runtime, or can be used as a springboard to attack other software running on the same or other machines is considered a security issue. As attacks we consider anything that affects the confidentiality, integrity and/or availability of the system.
+
+
+## What happens after the report?
+
+All security bugs in the Erlang/OTP distribution should be reported to [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). Your report will be handled by a small security team at the OTP team.
+
+Please use a descriptive email title for your report. After the initial response to your report, the security team will keep you updated on the progress and decision being made towards a fix and release announcement.
+
+## Flagging Existing Issues as Security-related
+
+If you believe that an existing public issue on [https://github.com/erlang/otp/issues](https://github.com/erlang/otp/issues) is security-related, we ask that you report it via [https://github.com/erlang/otp/security](https://github.com/erlang/otp/security). The title should contain the issue ID on [https://github.com/erlang/otp/issues](https://github.com/erlang/otp/issues) (e.g. Flagging security issue ERL-001). Please include a short description to motivate why it should be handled according to the security policy.
```

Comment:
what is ERL-001? some leftover from old days?

Reply:
I took an example from `xmerl`. Please let me know if it does not look good, or if it is not a good example.
If you have any preferred way of writing this, let me know with a suggestion :)

Reply:
ERL-001 is a sample identifier not used anymore? is it?
you intended to refer to github issue numer - maybe GH-001 is wanted?